src/main/java/io/strimzi/kafka/quotas/StorageChecker.java

```diff
@@ -77,13 +80,25 @@ public void run() {
         }
     }
 
+    @Deprecated(forRemoval = true, since = "0.3.0")
```

Comment:
I've left this in for API compatibility. 

Happy to delete if that is not needed.